revoltek/losoto
# Module

Packaging of losoto as a module. It can be installed using setup.py as indicated in issue #6 .

The main caveat is the change of the name of the losoto script from losoto.py to losoto. Using losoto.py produced a clash of names in the imports of the H5parm scripts (they tried to import from this file instead than from the module).

To test the changes you can:
```
git fetch
git checkout --track origin/module
vagrant up
vagrant ssh
cd losoto
sudo python setup.py develop
```
Please check that everything is OK before merging, I did some tests but not extensive.
